test/spec/modules/medianetBidAdapter_spec.js

```diff
@@ -566,17 +566,16 @@ describe('Media.net bid adapter', () => {
   });
 
   describe('slot visibility', () => {
-    let mock;
+    let sandbox = sinon.sandbox.create();
```

Comment:
You should just put the sandbox creation in the outermost `describe` block's `before` and the `restore` in its `after`.  Doing this multiple times in different nested-levels of `describe` make it difficult to see if the sandbox is being properly cleaned up.